free-courses-en.md

```diff
@@ -106,6 +106,7 @@
 * [Angular Fast Crash Course](https://www.udemy.com/angular-fast-crash-course/) - Edwin Diaz, Coding Faculty Solutions (Udemy)
 * [Angular for Beginners](https://www.udemy.com/course/angular-for-beginners-course/) - Angular University (Udemy)
 * [Angular Tutorial For Beginners](https://www.youtube.com/playlist?list=PLC3y8-rFHvwhBRAgFinJR8KHIrCdTkZcZ) - Codevolution (YouTube)
+* [Learn Angular - Full Tutorial Course](https://www.youtube.com/watch?v=2OHbjep_WjQ) - freeCodeCamp.org (YouTube)
```

Comment:
There is a playlist version of what appears to be the same material, could we use that instead?

```suggestion
* [Learn Angular - Full Tutorial Course](https://www.youtube.com/playlist?list=PLYxzS__5yYQlqCmHqDyW3yo5V79C7eaTe) - freeCodeCamp.org (YouTube)
```